config/initializers/assets.rb

```diff
@@ -8,7 +8,7 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w(misc/*.js email.css ie*.css)
+Rails.application.config.assets.precompile += %w(app/controllers/*.js misc/*.js email.css ie*.css)
```

Comment:
So should I remove the other namespaced JS folders as well?

To be clear, this file was in `app/assets/javascripts/app/controllers` not in the server-side folder